src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -147,6 +158,41 @@ export default class PatientViewMutationTable extends MutationTable<
                 PanelColumnFormatter.getGenePanelIds(GenePanelProps(d)),
         };
 
+        // customization for ASCN-related columns
+        // Patient view differs from default/results view because multiple samples can appear in a row
+        // due to same variant in multiple samples
+        // This can lead to cases where there are multiple icons/tooltips in a single cell
+        // therefore patient view needs sampleManager to indicate which values match which samples
+
+        this._columns[
+            MutationTableColumnType.CANCER_CELL_FRACTION
+        ] = getDefaultCancerCellFractionColumnDefinition(
+            this.getSamples(),
+            this.props.sampleManager ? this.props.sampleManager: undefined
```

Comment:
Do you need to check `sampleManager`? It can only be an instance of the sampleManger or undefined anyway